### PR TITLE
feat: use web instead of frontend

### PR DESCRIPTION
**Pull request by juzhiyuan**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
___
### New feature or improvement
- Describe the details and related test reports.

Use the web folder to keep the frontend's files.

### CODE REVIEW

**Comment by juzhiyuan** [comment]
`Deploy with Docker` is down due to it's using v2.0 as source codes(`/frontend/xx`), but we are using `/web/xxx`, just ignore this failure.

File: docs/develop.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ $ yarn start
 
 ### Add E2E test cases
 
-Please refer to [E2E Documentation](../frontend/src/e2e/README.md).
+Please refer to [E2E Documentation](../web/src/e2e/README.md).
```
Comment (ShiningRush) [inline docs/develop.md:41]: We would better rename all `frontend` to `web` , such as L22

**Review by ShiningRush — CHANGES_REQUESTED** [review]

File: docs/develop.md (hunk 1 above)
Comment (juzhiyuan) [reply docs/develop.md:41]: Let's rename them

**Comment by juzhiyuan** [comment]
Backend E2E failed, it's not related to this PR. I just created an issue to track this.

https://github.com/apache/apisix-dashboard/issues/675

**Comment by juzhiyuan** [comment]
All keywords are renamed to `web` according to @ShiningRush's suggestion, I'm going to merge this.

**Pull request merged**